app/views/posts/show.html.erb

```diff
@@ -1,15 +1,26 @@
 <div class="container mt-5">
-  
   <div class="card mb-4">
     <div class="card-body">
       <h4 class="card-title"><%= @post.title %> by <%= @post.author.name %></h4>
-      <div class="d-flex gap-1">
+      <div class="d-flex gap-1 justify-content-end">
         <p class="m-0">Comments: <%= @post.comments_counter %>,</p>
         <p class="m-0">Likes: <%= @post.likes_counter %></p>
       </div>
       <p class="mt-2"><%= @post.text %></p>
+
+      <%= form_for(@post.likes.new, url: user_post_likes_path(@post.author, @post)) do |f| %>
+        <%= f.hidden_field :author_id, value: current_user.id %>
+        <%= f.submit "Like", class: "btn btn-primary" %>
+      <% end %><br>
+
+      <%= form_for(@post.comments.new, url: user_post_comments_path(@post.author, @post)) do |f| %>
+      
+        <%= f.text_area :text, placeholder: "Leave a comment..." %>
```

Comment:
- [ Optional ] I think it would be nice if you made this field a required field. This will prevent users from submitting empty comments on a post. Currently, it's possible for a user to add empty comments to a post please see the screenshot below for more details on this and rectify it.

## Empty comments on a post
![Screenshot from 2023-08-26 20-13-24](https://github.com/musangamfure/Blog-app/assets/16392046/2ca77dfd-9406-43a7-b61e-37fde3bbeb36)